runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsOverlay.java

```diff
@@ -50,7 +51,9 @@
 	private static final int MAX_FPS = 50;
 	private static final int FPS_SIZE = MAX_FPS + 1;
 	private static final int Y_OFFSET = 14;
-	private static final int VALUE_X_OFFSET = 15;
+	private static int VALUE_X_OFFSET = 15;
+	private final PanelComponent panel = new PanelComponent();
```

Comment:
Move the panel component where it belongs, next to other instance field declarations

Reply:
Also, this is unused?